context-windows/src/context.ps1

```diff
@@ -1343,11 +1343,16 @@ If ( !(Test-Path "$ctxDir") ) {
     mkdir "$ctxDir"
 }
 
-# Move old logfile away - so we have a current log containing the output of the last boot
+# Delete .old log file (if exist) - simple rotation
+If (Test-Path "$ctxDir\opennebula-context-old.log") {
+    Remove-Item "$ctxDir\opennebula-context-old.log"
+}
+
+# Move last logfile away - so we have a current log containing the output of the last boot

```

Comment:
Also, why not update the comment to something like ...output of the last service run
But I'm not 100% sure with tis one.

Reply:
Not sure either. Both ways seems understandable for me. If someone from opennebula team decide to change these comments — I'm okay with that. So I'll leave it as that for now.